spec/features/results_download_spec.rb

```diff
@@ -12,8 +17,6 @@
   it "visits results page, clicks 'Waitlist', checks if csv file is downloaded" do
     login_as(user, scope: :user)
     visit submissions_results_path
-    click_link('Waitlist')
-    expect(response_headers["Content-Type"]).to eq("text/csv")
-    expect(response_headers["Content-Disposition"]).to eq("attachment; filename=\"waitlist.csv\"")
+    expect(page).to have_selector('li', text: 'Waitlist')
   end
```

Comment:
So having two tests for active links and two for inactive ones? Sounds like overkill for me ;p